### PR TITLE
Avatar and badge

### PR DESCRIPTION
**Pull request by JaanviNL**
Screenshot (6)
Screenshot (8)

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap');
+
+*
+{
+    --lightgrey:#e5e5e5;
+    font-family: 'Montserrat', sans-serif;
+}
+.avatar-container
+{
+    width: 600px;
```
Comment (anupkgurung) [inline components/avatar/avatar.css]: try to avoid px and use rem instead

File: components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    --dnd: #ef4444;
+    --lightgrey: #e5e5e5;
+
+}
+
+.badge-container {
+    width: 36rem;
+    background-color: var(--lightgrey);
+    border: 0.1rem solid #a3a3a3;
+    padding: 2rem;
```
Comment (anupkgurung) [inline components/badge/badge.css:17]: please move the hex code to root and use the variable instead.

File: components/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.online {
+    background-color: var(--online);
+
+}
+
+.offline {
+    background-color: var(--offline);
+}
```
Comment (anupkgurung) [inline components/badge/badge.css:50]: offline seems not very specific class name,  i assume this is a status so more specific class name could be used for improving readability. e.g.(status-on/online).

**Review by anupkgurung — APPROVED** [review]
mentioned few suggestions could implement that. Rest seems good !!

**Review by Creator-singh — APPROVED** [review]
Beside changes requested by Anup all good to go

**Review by maitrakhatri — APPROVED** [review]

**Pull request merged**